File: demo/fibonacci/fib.py
import sys

def fib(n):
    if n <= 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)


n = 10
if len(sys.argv) == 2:
    n = int(sys.argv[1])
assert(n > 0)
sum = fib(n)
print(sum)

File: demo/fibonacci/fib.java
class Fibo{
    public static int fib(int n)
        {
            if (n <= 1)
                return 1;
            else
                return fib(n - 1) + fib(n - 2);
        }
    public static void main(String args[])
    {
        int n = 10;
        System.out.print( fib(n) );
    }
}